src/OmniSharp.MiscellanousFiles/MiscellanousFiles.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Composition;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using OmniSharp.FileSystem;
+using OmniSharp.FileWatching;
+using OmniSharp.Mef;
+using OmniSharp.Models.WorkspaceInformation;
+using OmniSharp.MSBuild;
+using OmniSharp.Services;
+
+namespace OmniSharp.MiscellanousFiles
+{
+    [ExtensionOrder(After = ProjectSystemNames.MSBuildProjectSystem)]
```

Comment:
How does the interact with the legacy `DotNetProjectSystem` which handles old project.json projects?

Reply:
As of now we only await the MSBuild project system has completed the update, no interaction with the Dotnet Project System.

Reply:
Should we add the similar `IUpdates` interface and await the update for the DotnetProjectSystem ?